### PR TITLE
Refactor event certainty code base to follow new pattern 

### PR DESCRIPTION
**Pull request by Mloweedgar**

### CODE REVIEW

File: src/Events/EventCertainty/index.js [hunk 1]
```diff
@@ -1,24 +1,53 @@
...
+];
+
+const { confirm } = Modal;
+
 /**
  * @class
  * @name EventCertainties
- * @description Render Event Certainties list which have search box,
- * actions and event certainties list
+ * @description Render eventCertainty list which have search box, certainties and eventCertainty list
```
Comment (BenMaruchu) [inline src/Events/EventCertainty/index.js]: Please update `eventCertainty` to normal word like Event Certainties

File: src/Events/EventCertainty/index.js [hunk 2]
```diff
@@ -35,33 +68,61 @@ class EventCertainties extends Component {
...
+  openEventCertaintyForm = () => {
     openEventCertaintyForm();
   };
 
   /**
    * @function
-   * @name closeEventCertaintiesForm
-   * @description close event certainty form
+   * @name openEventCertaintyForm
+   * @description close eventCertainty form
```
Comment (BenMaruchu) [inline src/Events/EventCertainty/index.js]: here also

File: src/Events/EventCertainty/index.js [hunk 3]
```diff
@@ -35,33 +68,61 @@ class EventCertainties extends Component {
...
+  closeEventCertaintyForm = () => {
     closeEventCertaintyForm();
     this.setState({ isEditForm: false });
   };
 
   /**
    * @function
    * @name searchEventCertainties
-   * @description Search Event Certainties List based on supplied filter word
+   * @description Search EventCertainties List based on supplied filter word
```
Comment (BenMaruchu) [inline src/Events/EventCertainty/index.js:125]: Here also

File: src/Events/EventCertainty/index.js [hunk 4]
```diff
@@ -75,9 +136,9 @@ class EventCertainties extends Component {
   /**
    * @function
    * @name handleEdit
-   * @description Handle on Edit action for list item
+   * @description Handle on Edit certainty for list item
```
Comment (BenMaruchu) [inline src/Events/EventCertainty/index.js:139]: This should be `Edit Event Certainty`

**Review by BenMaruchu — CHANGES_REQUESTED** [review]
@Mloweedgar  I have left you some review, please update when you get time

**Review by navish — COMMENTED** [review]
Apart from @BenMaruchu comments, looks okay to me.

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**